kubernetes/env-create.md

```diff
@@ -0,0 +1,232 @@
+---
+category: Kubernetes
+expires: 2018-01-31
+---
+# Creating a Kubernetes Environment
```

Comment:
Same thing as my previous comment, I'd like to aim for us referring to the "Cloud Platform" rather than Kubernetes. So this would be creating a "Cloud Platform" environment.

Reply:
+1 - but we should probably have a section near the top which says the core of this is Kubernetes. The docs link out to the Kubernetes docs in various places, which makes sense, but I think it could seem a bit out of the blue without a brief preamble introducing Kubernetes